docs/Gemfile.lock

```diff
@@ -0,0 +1,53 @@
+GEM
```

Comment:
There's already https://github.com/GoogleChrome/sw-helpers/blob/master/Gemfile.lock

Is there a reason for this extra copy? Can you just move the old one if it needs to be at this location?